renderer/components/Charts/Arrows.tsx

```diff
@@ -1,48 +1,50 @@
 import React from "react";
-import { fmtSize } from "../../util/minor";
+let filesize: any;
+import("filesize")
+  .then((module) => {
+    filesize = module.filesize;
+  })
+  .catch((err) => console.error("Failed to load the module", err));
```

Comment:
no don't dynamically import either, are you trying to get around missing types error?

Reply:
enable allowSyntheticDefaultImports (by setting it to true) in tsconfig.json, and you'll be able to import it

Reply:
tried but still getting the same error

Reply:
github desktop
<img width="1440" alt="image" src="https://github.com/ArweaveTeam/arweave-miner-ui/assets/60331864/7d16e1aa-b05e-4fe2-974d-fc3d5c2078e0">

code
<img width="543" alt="image" src="https://github.com/ArweaveTeam/arweave-miner-ui/assets/60331864/cad29ebe-d746-4cd1-a6d4-2a32b2f3f4fc">

tsconfig
<img width="978" alt="image" src="https://github.com/ArweaveTeam/arweave-miner-ui/assets/60331864/b8a49377-f9df-450b-8ff6-912bb4a53e07">

Reply:
you may want to set this compilerOptions just in the root tsconfig, it should allow you to import cjs modules as default, let me try this...